### PR TITLE
Add "New Project" Button

### PR DESCRIPTION
**Pull request by arottersman**
## Overview

- Add a button the model stage's project header: "New Project" to clear the map and go back to draw
- update "Change Area" button in analyze to clear the AoI

Connects #1896 
Connects #1874 

### Demo

gu5mv2dvob

## Testing Instructions

 * Pull and bundle
 * In model stage check that the "new project" button takes you back to draw with the AoI cleared
 * In analyze stage check the "change area" button clears the AoI
 * Test that the header in model stage looks okay in IE11, FF, Safari, Chrome

### CODE REVIEW

**Comment by kellyi** [comment]
> In model stage check that the "new project" button takes you back to draw with the AoI cleared

This works, but/and it seems not to take me back to the geolocated map center & extent. Thought it was worth making sure this was intentional?

**Comment by kellyi** [comment]
> In analyze stage check the "change area" button clears the AoI

Likewise, this one keeps me at the extent & center of the AOI? Wondering if it's worthwhile to have the two back buttons have the same behavior?

**Comment by kellyi** [comment]
Tried this out Chrome, Safari, and FF on macOS & IE11 on Windows and the header appeared properly in each and the functionality seemed to work well. Wondering if we should address the map extent qs above? One way to handle it is to normalize the behavior for each: either each back button stays with the center & zoom from the prior AOI or it refocuses on the continental US?

**Comment by arottersman** [comment]
@jfrankl, @ajrobbins thoughts?

- I think we should fix it so the "New Project" button takes you back to the geolocated extent (if that's enabled)
- I think "Change Area" from analyze should take you back to where the AOI was (without the AOI on the map) for cases where you've found an area you're interested in but haven't gotten the AOI right

**Comment by jfrankl** [comment]
@arottersman that sounds perfect to me.

**Comment by arottersman** [comment]
@kellyi, 4063cfc should fix the geolocating not happening when you click "New Project"

**Comment by kellyi** [comment]
Not sure if it's caused by these changes or just revealed by them, but it seems like using the back button doesn't stop the polling & eventually this error results in the console:

screen shot 2017-06-12 at 5 56 10 pm

**Comment by kellyi** [comment]
(Doesn't seem breaking, btw).

**Comment by kellyi** [comment]
The re-geolocation thing works well, however! Going to check it out in IE11.

**Comment by kellyi** [comment]
Geolocation reset seems like it's not quite working in IE11, but I can't tell why. The correct JS code is there but it just sticks on the continental US map instead of zooming.

**Comment by kellyi** [comment]
Seems to be working well otherwise! The geolocation reset does work in Firefox and Chrome, just not in IE.

I believe the console error shown above is the same as https://github.com/WikiWatershed/model-my-watershed/issues/1716 so we should try to address that soon.

**Comment by arottersman** [comment]
- I think I'll look into that console error for #1860 —it'll touch related pieces of the model
- I haven't been able to get browserstack ie11's geolocate to work for staging or this branch. Trying to boot up the asus to test (can't run my windows VM and MMW at the same time)

**Comment by arottersman** [comment]
Asus is all charged up, and I think the geolocation is working okay there (feel free to take and test). There's a bit of a delay, but it does it eventually.

**Comment by kellyi** [comment]
Taking another look at this.

**Comment by kellyi** [comment]
Works well in Chrome, Firefox, and IE. Safari seems to block geolocation on localhost, so I couldn't test that but the rest of the functionality worked well.

I did notice that the breadcrumb navigation may get slightly out of sync when you click New Project from the modeling phase: the analyze step has been removed from the stack, but the word "Analyze" remains on the breadcrumb:

screen shot 2017-06-13 at 6 13 26 pm

However, looks like we're dropping that altogether in #1958, so probably worth ignoring here.

File: src/mmw/js/src/core/views.js [hunk 1]
```diff
@@ -346,13 +340,29 @@ var MapView = Marionette.ItemView.extend({
...
+        // Center the map on the U.S.
+        this._leafletMap.fitBounds([
+            [24.2, -126.4],
+            [49.8, -66.0]
+        ]);
+    },
+
+    /**
+    Attempt to locate the user and set the map to it
+    @param alwaysSetView -- override the map model setting `geolocationEnabled` (used to prevent
```
Comment (kellyi) [inline src/mmw/js/src/core/views.js]: I like this docstring style, so I'm in favor of keeping it even though the other method comments in this file are more terse.

File: src/mmw/js/src/modeling/templates/projectMenu.html [hunk 2]
```diff
@@ -1,46 +1,53 @@
...
-                <li role="presentation"><a role="menuitem" tabindex="-1" id="itsi-clone">Embed in ITSI</a></li>
-            {% endif %}
-            {% if user and not itsi_embed %}
-                <li role="separator" class="divider"></li>
-                <li role="presentation"><a role="menuitem" tabindex="-1" data-url="/projects/">My Projects</a></li>
+        </button>
+        {% if editable or is_new or itsi %}
+            <ul class="dropdown-menu menu-right" role="menu">
+                {% if not is_new and editable %}
+                    <li role="presentation"><a role="menuitem" tabindex="-1" data-toggle="modal" id="share-project">Share</a></li>
```
Comment (kellyi) [inline src/mmw/js/src/modeling/templates/projectMenu.html]: Consider breaking these element tags across several lines, one tag per line, to make this more legible.

**Comment by arottersman** [comment]
Thanks for all your thorough testing!

**Comment by arottersman** [comment]
@azavea-bot rebuild

**Pull request merged**